### ISSUE TITLE
Recruit and schedule participants for interviews (Web Components)

### ISSUE THREAD

**Issue by jaclinec** [uswds/uswds-team#382]
We're planning interviews with participants to inform Web Components work (see details in #381 ). Steps for recruiting are:

- [x] Identify potential participants to reach out to
- [x] Create outreach communications to potential participants
- [ ] Send outreach communications to potential participants
- [ ] Coordinate and schedule sessions

**Comment by annepetersen** [comment]
[Recruitment spreadsheet edited](https://docs.google.com/spreadsheets/d/1xBoCRIGDGg5uzJqaqK6KhaT6IUPE9936Lb03qKNvDw8/edit?gid=0#gid=0) and ready to go

**Comment by jaclinec** [comment]
Update: I've identified potential participants to reach out to and they are going to be sourced from a few places: 

1.  Those who left their contact info on the WC survey they filled out
2.  Developers who've already signed up on our federal research panel
3.  Targeted groups and individuals whom we think may be good to talk to

@annepetersen that recruitment comms spreadsheet linked above was for WC survey distribution (#384 ). [The email communications draft](https://docs.google.com/document/d/1KtxkajN0-hW6MNIAspl98fzjyuPwQ2Mz9P6naGLwXdA/edit?usp=sharing) for WC interview recruiting is ready for your review (Jayme already reviewed it). One thing - I wasn't sure who we were reaching out to at 18F so that section is blank - let's chat about that soon.

**Comment by jaclinec** [comment]
Outreach and scheduling has started and so far we have 4 sessions scheduled. Scheduling will continue over the next couple of weeks.

**Comment by jaclinec** [comment]
After a pause in this effort to focus on the Web Component landscape analysis, I've picked this back up and have been reaching out to various people from 18F projects centering around performance and resilience. I've also scheduled a meeting with the California Design System folks and we met with a Web Components industry expert last Monday. 

So far we've conducted 5 interviews and we have 3 upcoming scheduled. I'll keep reaching out and scheduling as we see the need (i.e. if there are still things we want to learn that we haven't gotten enough information about from current interviews).
